integration/exception-handling/src/main/java/com/baumeister/sndbx/exceptionhandling/domain/BookingManagement.java

```diff
@@ -0,0 +1,27 @@
+package com.baumeister.sndbx.exceptionhandling.domain;
+
+import com.baumeister.sndbx.exceptionhandling.domain.model.Booking;
+import com.baumeister.sndbx.exceptionhandling.general.exception.NotFoundException;
+import org.springframework.stereotype.Component;
+import org.springframework.util.ObjectUtils;
+
+@Component
+public class BookingManagement {
+
+  public Booking getBooking(Long id) {
+    ObjectUtils.isEmpty(id);
+    if(id == 404L) {
+      throw new NotFoundException("The element of id " + id + " could not be found");
+    }
+    return Booking.builder()
+        .id(1L)
+        .email("m.mustermann@mail.com")
+        .description("No gluten")
+        .numberOfSeats(2)
+        .build();
+  }
+
+  public Booking createBooking(Booking booking) {
```

Comment:
Why do we need this method? Remove from all layers?

Reply:
This method throws the ValidationException when called with incorrect bookingTO. I'll write a test for it.